### DISCUSSION TITLE
scrum prep Milestone 2 - 4

### DISCUSSION THREAD

**Discussion by MrL9000** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#135] [General]
Write your scrums here

**Comment by MrL9000** [comment]
Admittedly I finished most of my work early, the only thing I have left is 1 lecture topic task which I do plan to work on this friday, so there isn't much to report all things considered, other than the menu view I previously mentioned got accepted.

**Comment by JaseyAcevedo** [comment]
Been a heavy week got 3 projects dropped for this weekend working on ltt

**Comment by Janiel777** [comment]
Ayude a resolver los bugs que habían a la hora de crear announcements y trips desde el dashboard.

**Comment by Brandell13** [comment]
I met with the team discussing the minimum points that must be met at the end of Mileston and the tasks for next spring

**Comment by marcosarraiza** [comment]
This week I met up with my team, and divided and started work on a 2 person task.

**Comment by warm007** [comment]
This week, I attended the team meeting where we discussed our progress and upcoming tasks. I’ve been working on the assigned task #75, "Vehicle Refactoring Information". During this weekend, I plan to work on the other assigned code task #86, "Selected Ownership View," to stay on track with the project milestone.

**Reply by sebasagir** [reply]
made a lecture topic task, started analyzing domain concepts to make aggregates

**Comment by Yahid1** [comment]
This week I clarified Milestone Documentation tasks as well as discussed milestone grades.

**Comment by IvanierB** [comment]
This week I am working on coding and documentation tasks. I am also searching for a possible lecture topic task.

**Comment by Alej24** [comment]
Due to having been sick and having an exam, I haven't been able to focus on much from the project but I plan on working on it throughout this weekend and next week.

**Comment by juan-ortiz92** [comment]
This week, I was investigating problems regarding the insertion of data on the database, mainly notifications.

**Comment by rafa-quiles** [comment]
I didn't work on tasks this week, attended team meetings

**Comment by JoelTorres20** [comment]
I worked on my current task this week and reviewed class slides to work on my next lecture topic task.

**Comment by dAninAd1248** [comment]
I was too busy with 3 tests to do anything this week.

**Comment by AlexisTraversoPR** [comment]
This week I attended the team meeting and assigned myself a task to work on some documentation fixes.

**Comment by yamayrie** [comment]
This week I've been working on a Lecture Topic Task that will be done for this sprint

**Comment by diegorobles2** [comment]
This week I've been looking for a LTT

**Comment by dylanDeJesus21** [comment]
Attended team meeting, working on documentation and ownership view creation
